core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -479,11 +481,35 @@ class AuthorizerIntegrationTest extends KafkaServerTestHarness {
     this.consumers.head.position(tp)
   }
 
-  @Test(expected = classOf[TopicAuthorizationException])
+  @Test(expected = classOf[TestTimeoutException])
   def testOffsetFetchWithNoTopicAccess() {
```

Comment:
Can we please improve the test names to be more specific? For example, `testOffsetFetchWithNoDescribeTopicAccess` (if applicable).